src/layers/tile/infinite.rs

```diff
@@ -167,4 +215,51 @@ impl<'map> InfiniteTileLayer<'map> {
             .get_tile_data(x, y)
             .map(|data| LayerTile::new(self.map, data))
     }
+
+    /// Returns an iterator over different parts of this map called [`Chunk`]s.
+    ///
+    /// These **may not** correspond with the chunks in the TMX file, as the chunk size is
+    /// implementation defined (Check [`Chunk::WIDTH`], [`Chunk::HEIGHT`]).
+    ///
+    /// The iterator item contains the position of the chunk in chunk coordinates along with a
+    /// reference to the actual chunk at that position.
+    ///
+    /// This iterator doesn't have any particular order set.
+    ///
+    /// ## Example
+    /// ```
+    /// # use tiled::{Loader, LayerType, TileLayer};
+    /// use tiled::Chunk;
+    ///
+    /// # let map = Loader::new()
+    /// #     .load_tmx_map("assets/tiled_base64_zlib_infinite.tmx")
+    /// #     .unwrap();
+    /// # if let LayerType::TileLayer(TileLayer::Infinite(infinite_layer)) =
+    /// #     &map.get_layer(0).unwrap().layer_type()
+    /// # {
+    /// for (chunk_pos, chunk) in infinite_layer.chunks() {
+    ///     for x in 0..Chunk::WIDTH as i32 {
+    ///         for y in 0..Chunk::HEIGHT as i32 {
+    ///             if let Some(tile) = chunk.get_tile(x, y) {
+    ///                 let tile_pos = (
+    ///                     chunk_pos.0 * Chunk::WIDTH as i32 + x,
+    ///                     chunk_pos.1 * Chunk::HEIGHT as i32 + y,
+    ///                 );
+    ///                 println!("At ({}, {}): {:?}", tile_pos.0, tile_pos.1, tile);
+    ///             }
+    ///         }
+    ///     }
+    /// }
+    /// # } else {
+    /// #     panic!("It is wrongly recognised as a finite map");
+    /// # }
+    /// ```
+    #[inline]
+    pub fn chunks(&self) -> impl ExactSizeIterator<Item = ((i32, i32), ChunkWrapper<'map>)> + 'map {
+        let map: &'map crate::Map = self.map;
+        self.data
+            .chunks
+            .iter()
+            .map(move |(pos, chunk)| (*pos, ChunkWrapper::new(map, chunk)))
+    }
```

Comment:
I guess that a `get_chunk(&self, x: i32, y: i32) -> Option<ChunkWrapper<'map>>` would also be nice, for those who want to render only the visible chunks. Though, I guess there would need to be quite a large number of chunks before this is faster than filtering the iterator returned by `chunks`.